src/AppBundle/Entity/ProcurationRequest.php

```diff
@@ -273,6 +281,13 @@ class ProcurationRequest
     private $processedAt;
 
     /**
+     * @var bool
+     *
+     * @ORM\Column(type="boolean")
+     */
+    private $reminded = false;
```

Comment:
This could be a good idea but do we really want to support it, we could skip setting it to false or reset it to true to use it again when needed. The real use case is to initially program many.